adam/curriculum/preposition_curriculum.py

```diff
@@ -500,6 +520,54 @@ def _make_in_front_training(
     )
 
 
+def _make_near_training(
+    num_samples: int = 5, *, noise_objects: bool = True
+) -> Phase1InstanceGroup:
+    figure_0 = standard_object("ball", BALL)
+    figure_1 = standard_object("book", BOOK)
+    figure_2 = standard_object("dad", DAD)
+    ground_0 = standard_object("cookie", COOKIE)
+    ground_1 = standard_object("table", TABLE)
+    ground_2 = standard_object("person", PERSON)
+    speaker = standard_object("speaker", MOM, added_properties=[IS_SPEAKER])
+    addressee = standard_object("addressee", LEARNER, added_properties=[IS_ADDRESSEE])
```

Comment:
We can remove the speaker and addressee from this situation template as they aren't relevant to the `near` relationship.